### PR TITLE
feat(dedicated.carbon-calculator): add HPC in description

### PR DESCRIPTION
**Pull request by antonymarion**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-11705
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

Display the name of the HPC in the description for the carbon calculator PDF screen

### Translations

- [x] CMT-4793

### CODE REVIEW

**Review by darsene — DISMISSED** [review]

File: packages/manager/modules/carbon-calculator/src/dashboard/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -6,6 +6,6 @@
   "carbon_calculator_dashboard_carbon_balance_operations": "Opérations",
   "carbon_calculator_dashboard_carbon_balance_greenhouse_gas": "Émissions de gaz à effet de serre",
   "carbon_calculator_dashboard_carbon_balance_explanation": "OVHcloud inclut l'ensemble des sources émettrices de CO2eq, données fournies par la formule :",
-  "carbon_calculator_dashboard_billing_explanation": "Ce document comprend les émissions de gaz à effet de serre du mois précédent :<ul><li>de vos services <strong>infrastructures Baremetal</strong></ul></li>dont vous êtes le <strong>contact facturation</strong>.",
+  "carbon_calculator_dashboard_billing_explanation": "Ce document comprend les émissions de gaz à effet de serre du mois précédent :<ul><li>de vos services <strong>infrastructures Baremetal</strong></li><li>de vos services <strong>Hosted Privated Cloud</strong></li></ul>dont vous êtes le <strong>contact facturation</strong>.",
```
Comment (ghyenne) [inline packages/manager/modules/carbon-calculator/src/dashboard/translations/Messages_fr_FR.json:9]: I think that if you add a number of different services over time, it's not going to be possible to maintain them all in a single translation.
I think this translation key needs to be split.

Comment (antonymarion) [reply packages/manager/modules/carbon-calculator/src/dashboard/translations/Messages_fr_FR.json]: Done, you can review again

File: packages/manager/modules/carbon-calculator/src/dashboard/carbon-footprint/template.html [hunk 2]
```diff
@@ -1,8 +1,22 @@
...
+        ></p>
+        <ul
+            data-ng-repeat="serviceName in $ctrl.availableServices track by $index"
+        >
+            <li
+                data-translate="carbon_calculator_dashboard_billing_explanation_service"
+                data-translate-values="{ serviceName: serviceName }"
+            ></li>
+        </ul>
+        <br />
```
Comment (darsene) [inline packages/manager/modules/carbon-calculator/src/dashboard/carbon-footprint/template.html:15]: Why do you set `margin-bottom: 0` and add a `<br>` element ? Maybe you should just adapt your css definition

File: packages/manager/modules/carbon-calculator/src/dashboard/carbon-footprint/template.html [hunk 3]
```diff
@@ -1,8 +1,22 @@
 <div class="row-fluid">
-    <p
-        class="carbon-balance"
-        data-translate="carbon_calculator_dashboard_billing_explanation"
-    ></p>
+    <div class="carbon-balance mb-4">
```
Comment (darsene) [inline packages/manager/modules/carbon-calculator/src/dashboard/carbon-footprint/template.html:2]: Did you check the mobile view ?

Comment (antonymarion) [reply packages/manager/modules/carbon-calculator/src/dashboard/carbon-footprint/template.html:2]: Yes it's good with galaxy for instance

**Review by antonymarion — COMMENTED** [review]
done

**Review by antonymarion — COMMENTED** [review]
done

File: packages/manager/modules/carbon-calculator/src/dashboard/carbon-footprint/controller.js [hunk 4]
```diff
@@ -55,6 +63,11 @@ export default class CarbonFootprintCtrl {
   $onInit() {
     this.$asyncFetching = null;
     this.previousMonth = this.carbonFootprintService.computePreviousMonth();
+    this.availableServices = SERVICES_AVAILABLE.map((u) =>
+      this.$translate.instant(
+        `carbon_calculator_dashboard_billing_explanation_${u}`,
+      ),
+    );
```
Comment (ghyenne) [inline packages/manager/modules/carbon-calculator/src/dashboard/carbon-footprint/controller.js:70]:
```suggestion
    this.availableServices = SERVICES_AVAILABLE.map((service) =>
      this.$translate.instant(
        `carbon_calculator_dashboard_billing_explanation_${service}`,
      ),
    );
```
It's better to have a variable name

File: packages/manager/modules/carbon-calculator/src/index.scss [hunk 5]
```diff
@@ -4,10 +4,15 @@
...
 @import 'bootstrap4/scss/_grid.scss';
 
+.carbon-balance-spacings {
+  *:not(:last-child) {
+    margin-bottom: 0px;
+  }
+}
 .carbon-balance {
   ul {
+    margin-bottom: 5px;
```
Comment (darsene) [inline packages/manager/modules/carbon-calculator/src/index.scss:14]: As we're using bootstrap, use its own classes --> https://getbootstrap.com/docs/4.0/utilities/spacing/

File: packages/manager/modules/carbon-calculator/src/index.scss [hunk 6]
```diff
@@ -4,10 +4,15 @@
 @import 'bootstrap4/scss/_utilities.scss';
 @import 'bootstrap4/scss/_grid.scss';
 
+.carbon-balance-spacings {
+  *:not(:last-child) {
+    margin-bottom: 0px;
```
Comment (darsene) [inline packages/manager/modules/carbon-calculator/src/index.scss:9]: use bootstrap class instead

**Review by darsene — CHANGES_REQUESTED** [review]

**Review by ghyenne — APPROVED** [review]

**Review by darsene — APPROVED** [review]

**Review by ghyenne — DISMISSED** [review]

**Review by anooparveti — DISMISSED** [review]

**Review by darsene — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Pull request merged**